bindings/python/vara-feature/pybind_Feature.cpp

```diff
@@ -9,24 +10,112 @@
 namespace vf = vara::feature;
 namespace py = pybind11;
 
-void init_feature_module(py::module &M) {
+void init_feature_module_feature(py::module &M) {
   py::class_<vf::Feature>(M, "Feature")
       .def_property_readonly("name", &vf::Feature::getName,
                              R"pbdoc(The name of the feature.)pbdoc")
       .def("is_optional", &vf::Feature::isOptional,
-           R"pbdoc(`True` if the feature is optional.)pbdoc");
+           R"pbdoc(`True` if the feature is optional.)pbdoc")
+      .def(
+          "is_root", &vf::Feature::isRoot,
+          R"pbdoc(`True` if this is the root feature of a `FeatureModel`.)pbdoc")
+      //===----------------------------------------------------------------===//
+      // Children
+      .def(
+          "children",
+          [](vf::Feature &F) { return py::make_iterator(F.begin(), F.end()); },
```

Comment:
Iterators are not indexable, I don't think I need it but it's not intuitive.

Reply:
As we talk today, iterators are the better interface for the Cpp side but in python `list()` conversions work if one wants to use indexing.
It looks like this is the more pythonic way :laughing: